pkg/server/api/svid/v1/service.go

```diff
@@ -112,12 +112,18 @@ func (s *Service) MintX509SVID(ctx context.Context, req *svidv1.MintX509SVIDRequ
 	if err != nil {
 		return nil, api.MakeErr(log, codes.Internal, "failed to sign X509-SVID", err)
 	}
-	rpccontext.AuditRPCWithFields(ctx, logrus.Fields{
+
+	x509SVIDLogFields := logrus.Fields{
 		telemetry.SPIFFEID:  id.String(),
 		telemetry.DNSName:   strings.Join(csr.DNSNames, ","),
 		telemetry.Subject:   csr.Subject,
 		telemetry.ExpiresAt: x509SVID[0].NotAfter.Unix(),
-	})
+	}
+
+	rpccontext.AuditRPCWithFields(ctx, x509SVIDLogFields)
+	log.WithField(telemetry.Expiration, x509SVID[0].NotAfter.Format(time.RFC3339)).
```

Comment:
this log will have a duplicated value for expiration (ExpiresAt and Expiration)  but with different format, 
Just to avoid confusion, may you remove ExpiresAt from this log?

Reply:
I also noticed the discrepancy of field names/values and expected this to be a point of discussion 😄 . I thought it might be ok to have some duplication of fields between the audit log and general log since audit logging is disabled by default. This log has also existed for a long time, so I am hesitant to remove any fields to prevent breaking any users' log queries.  I think we have a few options here:

Another option could be to change the audit log field to match the existing general log key and value format `"expires_at": <unix time>` -> `"expiration": <RFC3339 time>`. This would apply to any other instances where `expires_at` is used instead of `expiration` as well. What do you think @MarcosDY ?

Reply:
but this "new" log entry will looks like:
```
{
     "message": "Signed X509 SVID",
     "data": [ 
          "spiffe_id": "spiffe://example.org/workload,
          "dns_name": "dns1,dns2",
          "subect": "subject",
          "expires_at": "1234",
          "expires": "<RFC3339 time>"
     ]    
}
```
So just removing "expires_at" from the list you use to create that log will keep the same log that we used to have on CA,
and of course we must keep `expire_at` on audit log

Reply:
Ohh ok I see your point now, this one place has both fields in the new log. Yes, I can remove `expires_at` in this instance. Good catch!